admin/sitemodselectrical.md

```diff
@@ -17,7 +17,7 @@ HSBNE Inc. has certain restrictions that apply at our current property. This pol
 # Area Downtime
 HSBNE Members expect that the site is available for use 24/7 as explained during their induction and on our website. This policy aims to avoid the dissapointment of showing up on site to find an area that is unable to be used. This is especially important for members who have to travel far to get to HSBNE or who make special arrangements/plans.
 
-* Members must be notified in writing of all changes to an area of HSBNE that may, or will, cause Area Downtime.
+* The member in charge of any modification to a HSBNE area must make an anouncement via the official communication method of all changes that may, or will, cause Area Downtime.
```

Comment:
Is this the person performing the change or is it the cause leader for that area? This is the sort of thing a RACI is made for. Do we need to provide provisions for downtimes caused by third parties? ie the exec isnt in charge of say, water works performed by cbre, but we try and pass on the notification to members, typically by email.